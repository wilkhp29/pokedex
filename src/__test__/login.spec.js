import React from 'react';
import { shallow } from 'enzyme';
import Login from '../Views/login';

it('renderizando a Login', () => {
    const wrapper = shallow(<Login />);

    expect(wrapper).toMatchSnapshot();
});