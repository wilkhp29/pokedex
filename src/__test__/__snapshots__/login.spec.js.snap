// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderizando a Login 1`] = `ShallowWrapper {}`;
